tiledb/sm/query/query.cc

```diff
@@ -1475,6 +1489,12 @@ Status Query::check_buffer_names() {
 
 Status Query::create_strategy(bool skip_checks_serialization) {
   if (type_ == QueryType::WRITE || type_ == QueryType::MODIFY_EXCLUSIVE) {
+    // Set the fragment timestamp if it has not yet been set.
```

Comment:
`create_strategy` is called from serialization code as well (`query_from_capnp`) but in this PR we don't serialize those new fields (`current_fragment_timestamp_` or `default_fragment_timestamp_`).
I am a bit out of context here, so I don't know if they only apply to Dimension Labels and maybe we don't support serialization for those just yet? But let's double check this won't break creating query strategies for the rest of scenarios as well. @KiterLuc what do you think?

Reply:
@ypatia I think we lack automated REST test coverage to comfortably make those changes.

Reply:
If I understand the serialization correctly (a big IF), the `default_fragment_timestamp_` does need to be added to serialization, but it is only used by dimension labels. The `current_fragment_timestamp_` doesn't need to be serialized, because it will only be used on the remote query and will be initialized when the query is initialized.